build.sh

```diff
@@ -10,36 +10,40 @@ path_orbital=`pwd`
 function build_bios() {
   cd ${path_orbital}
   cd ${path_bios}
-  make -j4
+  make -j$(nproc)
   mv out/bios.bin ../${path_bin}/ubios.bin
 }
 
 function build_grub() {
   # Dependencies
-  sudo apt-get -qq install python
-  sudo apt-get -qq install dh-autoreconf
-  sudo apt-get -qq install bison flex
+  if [ -x "$(command -v apt)" ]; then
+    sudo apt-get -qq install python \
+                             dh-autoreconf \
+                             bison flex
+  fi
   # Building
   cd ${path_orbital}
   cd ${path_grub}
   ./autogen.sh
-  ./configure --target=x86_64
-  make -j4
+  ./configure --target=x86_64 --disable-werror
```

Comment:
What's the reasoning behind adding `--disable-werror`

Reply:
GCC 8 added some warnings which our fork of GRUB hasn't addressed, not sure what upstream is like though.

Reply:
Fair enough, thanks for the patch.